File: Java/opentraceability/src/main/java/opentraceability/queries/DigitalLinkQueryOptions.java
package opentraceability.queries;

import opentraceability.mappers.EPCISDataFormat;
import opentraceability.models.events.EPCISVersion;

import java.net.URI;

public class DigitalLinkQueryOptions {
    public URI url = null;

    public EPCISVersion version = EPCISVersion.V2;

    public EPCISDataFormat format = EPCISDataFormat.JSON;
    public boolean EnableStackTrace = false;
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/queries/digitallinkqueryoptions.kt
package opentraceability.queries

import opentraceability.mappers.EPCISDataFormat
import opentraceability.models.identifiers.*
import opentraceability.models.events.*
import opentraceability.models.events.EPCISVersion
import java.net.URI

class DigitalLinkQueryOptions {
    var URL: URI? = null
    var Version: EPCISVersion = EPCISVersion.V2
    var Format: EPCISDataFormat = EPCISDataFormat.JSON
    var EnableStackTrace: Boolean = false
}
